examples/platform/linux/AppMain.cpp

```diff
@@ -571,6 +574,12 @@ void ChipLinuxAppMainLoop(AppMainLoopImplementation * impl)
     };
     otherDelegate = &energyEvseTestEventTriggerDelegate;
 #endif
+#if CHIP_DEVICE_CONFIG_ENABLE_ENERGY_REPORTING_TRIGGER
+    static EnergyReportingTestEventTriggerDelegate energyReportingTestEventTriggerDelegate{
```

Comment:
This has stopped scaling, in that one handler can override another. This should not be in the main, it should be lined-up in  the sample's main.cpp.

Reply:
Do we need to raise a follow-up Issue to address this (post TE2?)

Reply:
After 1.3 (SVE) we will start enforcing this kind of things - no more ifdef-driven code and unit testing. Followup for now because 1.3.

Reply:
Created a follow up: https://github.com/project-chip/connectedhomeip/issues/31716